var/spack/repos/builtin/packages/geant4-data/package.py

```diff
@@ -195,6 +193,19 @@ class Geant4Data(BundlePackage):
         for _d in _dsets:
             depends_on(_d, type=("build", "run"), when=_vers)
 
+    _datasets_tendl = {
+        "11.0:11.3": "g4tendl@1.4",
+        "10.4:10.7": "g4tendl@1.3.2",
+        "10.3:10.3": "g4tendl@1.3",
+    }
+
+    variant("g4tendl", default=True, when="@10.3:", description="Enable G4TENDL")
+    with when("+g4tendl"):
+        for _vers, _d in _datasets_tendl.items():
+            depends_on(_d, type=("build", "run"), when="@" + _vers)
+    variant("g4nudexlib", default=True, when="@11.3.0:11.3", description="Enable G4NUDEXLIB")
+    variant("g4urrpt", default=True, when="@11.3.0:11.3", description="Enable G4URRPT")
```

Comment:
The only comment (which I'd support others weighing in) is that it might be a little cleaner to leave off the `g4` from the variant names. The full name will show up in the spec and environment variables.

Reply:
(OT: `geant4-data +nudexxxlib` would be a spec that raises DOE concerns.)